odoo/addons/test_new_api/tests/test_domain.py

```diff
@@ -52,15 +52,15 @@ def test_empty_char(self):
         self.assertListEqual(EmptyChar.search([('name', 'ilike', 'name')]).mapped('name'), ['name'])
         self.assertListEqual(EmptyChar.search([('name', 'not ilike', 'name')]).mapped('name'), ['', False])
 
-        self.assertListEqual(EmptyChar.search([('name', '=', '')]).mapped('name'), [''])
+        self.assertListEqual(EmptyChar.search([('name', '=', '')]).mapped('name'), ['', False])
```

Comment:
This is terrible 😨
This would make much, much, more sense and make the life of devs easier.

```suggestion
        self.assertListEqual(EmptyChar.search([('name', '=', '')]).mapped('name'), [''])
        self.assertListEqual(EmptyChar.search([('name', '=', False)]).mapped('name'), [''])
```

Reply:
We will consider that null is equal to '' and returns both null and '' values. We will also change the read function so that False is never returned and converted directly to '', it will be easier for everyone.